pulp_container/app/modelresource.py

```diff
@@ -95,7 +97,7 @@ def set_up_queryset(self):
 
     class Meta:
         model = Tag
-        import_id_fields = model.natural_key_fields()
+        import_id_fields = tuple(chain.from_iterable(model._meta.unique_together))
```

Comment:
Leave a comment about why this needs to be done

Reply:
Actually, this should not be true when relying on the natural_key PR in pulpcore. It purposefully does _not_ change `natural_key_fields`.

Reply:
yeah, sorry i have a typo s/contains/constraints/

Reply:
I'm a mess, got confused with natural_keys()